libqtile/core/manager.py

```diff
@@ -775,7 +775,11 @@ def get_interesting_pid(win):
             # managed and don't have any state.
             if not isinstance(win, window.Window):
                 return None
-            return win.window.get_net_wm_pid()
+            try:
+                return win.window.get_net_wm_pid()
+            except Exception:
```

Comment:
Why not catching `xcffib.ConnectionError`? Can other exceptions be raised?

Reply:
I didn't notice other exceptions were raised, but I think it's ok to mute others so that it can gracefully shutdown?